examples/aws/poc/dsf_deployment/variables.tf

```diff
@@ -246,7 +246,7 @@ variable "agentless_gw_hadr" {
 
 variable "hub_instance_type" {
   type        = string
-  default     = "r6i.xlarge"
+  default     = "r6i.2xlarge"
```

Comment:
You should also change these in the sonar base module. The reason we put relatively small instances is to make it cheaper for poc environments which aren't being used by customers.

Reply:
on it.